cmd/deploy/start.go

```diff
@@ -41,6 +47,13 @@ type FormattableDeployStartResponse struct {
 	err             error
 }
 
+type ArmoryDeployClient interface {
+	PipelineStatus(ctx context.Context, pipelineID string) (*de.PipelineStatusResponse, *_nethttp.Response, error)
```

Comment:
I never understood why it was `_nethttp` in here - can we rid of that? (`_nethttp` -> `nethttp`)

Reply:
Fixed